packages/discord.js/package.json

```diff
@@ -59,8 +59,11 @@
   "devDependencies": {
```

Comment:
For a future PR, shouldn't we move all dev deps to the root of the project? CC @iCrawl